cmd/run.go

```diff
@@ -51,13 +51,13 @@ func (r *RunConfig) flags() []cli.Flag {
 	return []cli.Flag{
 		&cli.StringFlag{
 			Name:        "lang",
-			Usage:       "SDK language to run ('go' or 'java' or 'ts')",
+			Usage:       "SDK language to run ('go' or 'java' or 'ts' or 'py')",
 			Required:    true,
 			Destination: &r.Lang,
 		},
 		&cli.StringFlag{
 			Name:        "version",
-			Usage:       "SDK language version to run. Typescript versions may start with `/` to use a local SDK via an absolute path.",
+			Usage:       "SDK language version to run. Most languages support versions as paths.",
```

Comment:
Does path have to be absolute?

Reply:
Hrmm, I'll have to check. For Python I do a simple glob which would work for cwd but I don't seem to make it absolute when putting in the pyproject.toml. I will confirm.

Reply:
I was not accepting relative paths, but I do now. There is a weird bug with Poetry where an absolute path on Windows had to have the drive letter capitalized, so I did that too.